cybershell/plugins.py

```
@@ -0,0 +1,52 @@
+class FormDiscoveryPlugin(PluginBase):
+    name = 'FormDiscoveryPlugin'
+    def run(self, **kwargs) -> PluginResult:
+        target = kwargs.get('target','')
+        if not self.in_scope(target):
+            return PluginResult(self.name, False, {'reason':'out_of_scope'})
+        forms = [{'action': '/login', 'method': 'POST', 'inputs': ['username','password']}]
+        return PluginResult(self.name, True, {'target': target, 'forms': forms, 'notes': 'discovery simulated'})
+
+class HeuristicAnalyzerPlugin(PluginBase):
+    name = 'HeuristicAnalyzerPlugin'
+    def run(self, **kwargs) -> PluginResult:
```

Ruff (0.12.2)

Missing type annotation for `**kwargs`

(ANN003)